APIProject/DetailsView/DetailViewController.swift

```diff
@@ -7,19 +7,24 @@
 
 import UIKit
 import SDWebImage
-class DetailViewController: UIViewController{
+
+final class DetailViewController: UIViewController{
```

Comment:
Dar un espacio antes de la llave de apertura 
```suggestion
final class DetailViewController: UIViewController {
```